src/auth/index.js

```diff
@@ -13,9 +13,17 @@ const EMBEDDED_USER_AUTH_DATA_PARTS = 4;
 
 export default class Auth {
   constructor(client, rootUrl, options) {
+    let namespace;
+    if (client === null || client.clientAppId === '') {
```

Comment:
nit: i think `if(!client || !client.clientAppId){ ... ` here might be a little more robust